task03-one-piece-website/onepiece.html

```diff
@@ -0,0 +1,192 @@
+                </div> 
+              </div> 
+            </div>
+          </nav>
+    </header>
+    <div class="main-photo-info-container">
+      <div class="one-piece-container">
+      <img class="img-one-piece-group" src="../images/one-piece-group.png">
+      <img class="img-second-logo" id="img-second-logo" src="../images/one-piece-logo.png">
+      </div>
+      <div class="sinopsis-container">
+        <div class="sinopsis-texts-container">
+          <h1 class="sinopsis-title">Sinopsis</h1>
+          <p class="sinopsis-text">Dawn Island is an island located in the East Blue and where the Goa Kingdom is situated.<br><br>
+          It is the home island of Monkey D. Luffy and the starting point of his journey into piracy, as well as the home island of his father, Monkey D. Dragon.
+          </p>
+          <span style="display:none" class="sinopsis-text-two" id="sinopsis-text-two"><br>This is also the adopted home of Portgas D. Ace after the death of his mother and the home of Sabo and his family.<br> <br>Reverse Mountain is an extremely tall mountain at one of the points where the four Blues meet and is one of the two known entrances to the Grand Line.<br><br> It is located in the Red Line and is the geographical antipode of Mary Geoise.</span>
+        </div>
+        <div class="sinopsis-read-all">
+          <span class="icon-angle-down" id="read-all-icon"></span>
+          <p type="button" class="read-all-btn" id="read-all-btn" onclick="readAll()" >Read All</p>
+        </div>
+      </div>
+    </div>
+    <div class="main-info-container">
+      <form>
+        <div class="col-12">
+          <div class="input-group w-90 mx-auto">
+              <div class="input-group-prepend">
+                  <span class="input-group-text"><i class="icon-search" ></i></span>
+              </div>
+              <input type="text" class="form-control search-form-control" placeholder="Search Characters, Islands or Objects">
+          </div>
+        </div>
+      </form>
+      <div class="main-photos-container">
+        <div class="characters-container">
+          <div class="titles-container">
+            <p class="section-titles">Characters</p>
+            <a href="/characters" class="see-all-titles">See all</a>
+          </div>
+          <div class="characters-photo-container">
+            <div class="characters-photo-item" id="img-monkey">
+               <p class="characters-photo-text">Monkey D. Luffy</p>
+            </div>
+            <div class="characters-photo-item" id="img-portgas">
+                <p class="characters-photo-text">Portgas D. Ace</p>
+            </div>
+            <div class="characters-photo-item" id="img-roronoa">
+               <p class="characters-photo-text">Roronoa Zoro</p>
+            </div>
+            <div class="characters-photo-item" id="img-nico">
+               <p class="characters-photo-text">Nico Robin</p>
+            </div>
```

Comment:
There's a lot of repeated code here, I'll give a tip, let's try to add this kind of repetitive code with JS, you can create the objects from there and inject them into the HTML, for example: select the parent container and create the inner HTML of these 4 photo items, if you need more help you can send a message in Slack.

Reply:
Understood, Pame said that for now I can leave it like that because there's a JS task after, so I can add this tip later in that task, I'll put a reminder when I fix this:)